src/utils/d3.ts

```diff
@@ -75,6 +74,40 @@ export const drawDots = <T>({
   }
 }
 
+export const drawLines = ({
+  svg,
+  className,
+  x1,
+  y1,
+  x2,
+  y2,
+  strokeWidth,
+  testId
+}: {
+  svg: d3.Selection<SVGGElement, unknown, null, undefined>
+  className: string
+  x1: number
+  y1: number
+  x2: number
+  y2: number
+  strokeWidth: number
+  testId?: string
+}): void => {
+  const line = svg
+    .append('line')
```

Comment:
It works - so I'm not requesting a change here - but I do wonder if passing in an array wouldn't be nicer? Don't know if it would be any faster - i.e. https://github.com/d3/d3-shape/blob/v2.0.0/README.md#_line